spring-cloud-gcp-samples/spring-cloud-gcp-bigquery-sample/README.adoc

```diff
@@ -10,6 +10,8 @@ This application allows you to load CSV data into BigQuery tables through either
 
 == Running the sample
 
+image:http://gstatic.com/cloudssh/images/open-btn.svg[link=https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fspring-cloud-gcp&cloudshell_open_in_editor=spring-cloud-gcp-samples%2Fspring-cloud-gcp-bigquery-sample%2FREADME.adoc]
```

Comment:
Since this opens in the project root, I think we need to tweak the instructions to be able to properly run the sample app. Maybe first do `./mvnw clean install` from project root, then cd into `spring-cloud-gcp-samples/spring-cloud-gcp-bigquery-sample`. And then you can follow the instructions.

Reply:
Added these two steps at the start of the instructions list

Reply:
Looks good, thanks! Feel free to proceed with the others in the same manner.